docs/contributing.md

```diff
@@ -105,3 +105,21 @@ Then:
 ```
 Click the link it outputs. As you save changes to files modified in your editor,
 the browser will automatically show the new content.
+
+
+How to File an Issue
```

Comment:
I think we can remove this from `contributing.md` and only have it in `README.md`